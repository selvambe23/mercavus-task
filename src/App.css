.App {
  text-align: center;
}

.ReactTable .rt-th, .ReactTable .rt-td {
  white-space: normal;
}

.header {
  padding: 20px;
  background: #333;
  color: #fafafa;
  margin-bottom: 25px;
}
.header-content {
  max-width: 960px;
  margin: auto;
  display: flex;
  align-content: center;
  justify-content: space-between;
}
.header .button {
  margin-bottom: 0.5em;
  border-bottom: 1px solid #333;
  background: #fafafa;
  color: #333;
  font-size: 1em;
  font-weight: bold;
  padding: 14px 25px;
  border-radius: 10px;
  text-decoration: none;
  display: inline-block;
}
