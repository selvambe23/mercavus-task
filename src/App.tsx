import React from "react";
import UserHobbies from "./Components/UserHobbies/UserHobbies";
import Header from './Components/Header/Header';
import "./App.css";

const App: React.FC = () => {
  return (
    <div className="App">
        <Header />
        <UserHobbies />
    </div>
  );
};

export default App;
