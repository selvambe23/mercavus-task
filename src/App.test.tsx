import React from 'react';
import { shallow } from 'enzyme';
import * as renderer from "react-test-renderer";
import App from './App';

describe('App Component', () => {
  it('renders snapshots, too', () => {
    // const wrapper = shallow(<App />);
    // expect(wrapper);
    // expect(wrapper.find('.App')).toBeDefined();

    const tree = renderer
      .create(<App />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
