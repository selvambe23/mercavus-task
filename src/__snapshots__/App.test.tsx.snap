// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders snapshots, too 1`] = `
<div
  className="App"
>
  <div
    className="header"
  >
    <div
      className="header-content"
    >
      <img
        alt="logo"
        src="https://cdn.shopify.com/s/files/1/0003/1938/9747/files/Mercavus-Logo-White_1024x1024.png?v=1539856605"
      />
    </div>
  </div>
  <div
    className="ReactTable"
  >
    <div
      className="userHobbyDetails"
    >
      <div
        className="userDetails"
      >
        <h1>
          User Lists
        </h1>
        <div
          className="userAddForm"
        >
          <div
            className="userAddform"
          >
            <input
              className="nameBox"
              onChange={[Function]}
              type="text"
              value=""
            />
            <button
              onClick={[Function]}
            >
              Add
            </button>
          </div>
        </div>
        <div
          className="userLists"
        />
      </div>
    </div>
  </div>
</div>
`;
